how_to_ask_for_help.html

```diff
@@ -1,46 +1,78 @@
-            <h2>Be aware of the following super handy resources to catch simple errors</h2>
-            <p><a href="https://validator.w3.org/">W3C website validator</a>: putting your page in here will let you know about many simple errors, such as unclosed tags.</p>
-            <p><a href="https://text-compare.com/">Text Compare</a>: Works like the diff built into codecademy. When you cannot see where you have made an error, and you are sure it looks like an example. This tool is useful for seeing where easily missed mistakes may be present.</p>
-        </li>
-    </ol>
-</body>
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">        
+        <link rel="stylesheet" href="./assets/css/style.css">
+        <link rel="stylesheet" href="assets/css/bulma_styles.css">
+        <link rel='icon' href='favicon.ico' type='image/x-icon'>
+        <script src="assets/scripts/script.js" defer></script>
+        <title>Asking for Help</title>
+    </head>
+    <body class="has-navbar-fixed-top">
+        <header>
+            <nav class="navbar has-shadow has-background-primary is-fixed-top">
+                <div class="navbar-brand">
+                    <a class="navbar-item" href="index.html"><img src="assets/img/codecademy_logo.png" alt="The codecademy logo" class="logo"></a>
+                    <a role="button" class="navbar-burger burger" aria-label="menu" aria-expanded="false" data-target="navbar-items">
+                        <span aria-hidden="true"></span>
+                        <span aria-hidden="true"></span>
+                        <span aria-hidden="true"></span>
+                    </a>
+                </div>
+                <div class="navbar-menu" id="navbar-items">
+                    <div class="navbar-end">
+                        <a class="navbar-item" href="./index.html">Home</a>
+                        <a class="navbar-item" href="./staff.html">Server Staff</a>
+                        <a class="navbar-item" href="./contact.html">Contact Us</a>
+                        <a class="navbar-item" href="./sharing_code.html">Sharing Code</a>
+                    </div> 
+                </div>
+            </nav>
+        </header>
+        <main class="container has-text-centered">
+            <h1 class="is-size-1">How to get Coding Help</h1>  
+            <p class="p-3">This is a guide on how to get help when you run into a problem while coding.</p>
+            <p class="p-3">Most people are very keen to offer help if they can, but we have noticed a few problems people make that mean it is harder for others to help them. This might cause you to get frustrated, and not get the help you need.</p>
+            <article class="container content has-background-white p-6 mt-4">
+                    <section>
+                        <h2 class="is-size-2">1. Let us see your code</h2>
```

Comment:
The font weight for this element is 600, but the headings on the other pages have a font weight of normal, can we update this to be consistent with the other pages?